ui/src/treeDataProvider/CreatingTreeItem.ts

```diff
@@ -26,8 +26,8 @@ export class CreatingTreeItem implements IAzureTreeItem {
 
     public get iconPath(): { light: string, dark: string } {
         return {
-            light: path.join(__filename, '..', '..', '..', 'resources', 'light', 'Loading.svg'),
-            dark: path.join(__filename, '..', '..', '..', 'resources', 'dark', 'Loading.svg')
+            light: path.join(__filename, '..', '..', '..', '..', 'resources', 'light', 'Loading.svg'),
```

Comment:
I don't this file or any of the image files moving. Why is there an extra `..`  in the path?

Reply:
I had to refactor the folder structure of build output a little bit so that I could get unit tests working. See the changes in tsconfig.json